Automattic/jetpack
# Jetpack: add post-publish-qr-post-panel block editor plugin

This PR adds the scaffolding for the QRPost panel plugin

Fixes https://github.com/Automattic/jetpack/issues/23275

#### Changes proposed in this Pull Request:

* Jetpack: add post-publish-qr-post-panel block editor plugin

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Enable [Beta extensions](https://github.com/Automattic/jetpack/tree/master/projects/plugins/jetpack/extensions#beta-extensions)
* Create a new post
* Click on `Publish` button
* The pre-publish panel will be open. Publish the post
* Confirm you see the QR Post panel there.

https://user-images.githubusercontent.com/77539/157131838-92f60dfd-cae5-4cf9-a0d4-044d4d197e60.mov
